### DISCUSSION TITLE
Dostawienie do Api godzin granicznych

### DISCUSSION THREAD

**Discussion by Agartallegro** [allegro/allegro-api#9305] [Suggestions - rejected]
https://allegro.pl/moje-allegro/sprzedaz/ustawienia-granicznych-godzin-wysylki Prośba o dostawienie do Api godzin granicznych

**Comment by emesio** [comment]
Podbijam wątek - proszę o rozważenie zarządzanie godzinami granicznymi przewoźników (to co w ordering-time-settings) przez API.
Odczyt aktualnej wartości / update nowej wartości?

Przewoźników jest coraz więcej (dochodzą kolejni w Delivery i poza Delivery, każdego trzeba aktualizować oddzielnie), w przypadku małych firm które muszą reagować dynamicznie na godziny podjazdów kurierów, aktualizacja tego przez stronę robi się kłopotliwa (do tego UI strony do zarządzania godzinami jest dramatycznie nieintuicyjne).

**Reply by PrzemyslawLukanowski** [reply]
Dziękuję za dodatkowy komentarz. Przekazałem temat ponownie do analizy.

**Comment by PrzemyslawLukanowski** [comment]
Sugestię musimy odrzucić. Aktualnie pracujemy nad zmianami w samym ustawianiu godzin granicznych, dlatego nie będziemy podejmować teraz prac nad wystawieniem dedykowanego API.
